subprojects/common/result/src/main/kotlin/com/avito/android/Result.kt

```diff
@@ -13,10 +13,6 @@ import com.avito.composite_exception.CompositeException
  */
 sealed class Result<T> {
 
-    abstract operator fun component1(): T?
```

Comment:
This API bad because it pushes users to check manually what is null and that leads to bugs